servertester/testsuites/cs531a5.py

```diff
@@ -287,7 +287,7 @@ def test_cgi_env_query_str(self, report):
         self.check_mime_is(report, "text/html")
         self.check_header_is(report, "Transfer-Encoding", "chunked")
         self.check_header_absent(report, "ETag")
-        self.check_payload_contains(report, "QUERY_STRING = var1=foo&var2=bar", "HTTP_USER_AGENT = CS 531-F18 A5 automated Checker")
+        self.check_payload_contains(report, "QUERY_STRING = var1=foo&var2=bar", "HTTP_USER_AGENT = {}".format(self.USERAGENT))
```

Comment:
Thanks for the catch. I prefer using f-string when possible, so if you could accept this suggestion, I will merge the PR and redeploy the tester server.

```suggestion
        self.check_payload_contains(report, "QUERY_STRING = var1=foo&var2=bar", f"HTTP_USER_AGENT = {self.USERAGENT}")
```